### PR TITLE
chore: update blockly-umd to use JSON toolbox

### LINKED ISSUE #12: Blockly-samples #1253
https://github.com/google/blockly-samples/issues/1253

### LINKED ISSUE #1253: Update blockly-umd to use JSON toolbox definitions instead of xml
**Category**
 - Examples

**Component**

blockly-umd

**Is your feature request related to a problem? Please describe.**

Many of the examples and plugins use XML toolboxes instead of the new JSON API. We should have our example code use the new APIs to best model what other developers can do with Blockly.

**Describe the solution you'd like**

Update the code in blockly-samples to use the JSON toolbox API.

**Additional context**

To work on this issue, first read the [background information](https://developers.google.com/blockly/guides/contribute/samples) about how to contribute to samples and verify your changes.

1. Learn about [defining a toolbox](https://developers.google.com/blockly/guides/configure/web/toolbox) and read about the XML format vs the JSON format.
2. In blockly-samples, navigate to the blockly-umd example's directory. Some examples can be started with `npm run start` and some can be opened directly by opening the `index.html` file in a browser. The README may give specific instructions, or you can look in the `package.json` to see what scripts are defined. Note you may need to run `npm install` in the example directory. Either way, open the example's demo page to observe the toolbox and its contents.
3. Write a new copy of the toolbox definition in the JSON format instead of the XML format. Where exactly it should be declared will depend on the example. It will most likely make sense to declare the toolbox definition close to the location where it is used, i.e. where `Blockly.inject` is called.
4. Refresh the demo page and make sure the toolbox looks the same as it did before. You can compare against the version in master on the [samples site](https://google.github.io/blockly-samples/) to make sure the behavior hasn't changed. Note that not all demos are published on the samples site so it may not be there.
5. Follow the rest of the instructions in the Contributing docs to verify your change and open a PR.

### PR DESCRIPTION
**Pull request by richardla4**
Description

- Updated examples/blockly-umd
- Rewrote the toolbox in JSON format

Fixes https://github.com/google/blockly-samples/issues/1253
Closes https://github.com/team-vibe-labs/blockly-samples/issues/12

### CODE REVIEW

File: examples/blockly-umd/index.js [hunk 1]
```diff
@@ -23,7 +23,75 @@
 
 document.addEventListener("DOMContentLoaded", function () {
 
-    var toolbox = document.getElementById('toolbox');
+    const toolbox = {
```
Comment (BeksOmega) [inline examples/blockly-umd/index.js]: Nit: Indentation should only be by two spaces for things that aren't line continuations. (You may want to check your IDE settings for a way to modify this)

Please change this to  look like:

```
const toolbox = {
  kind: 'flyoutToolbox',
  contents: [ /* etc... */ ],
}
```

File: examples/blockly-umd/index.js [hunk 2]
```diff
@@ -23,7 +23,75 @@
...
+                            },
+                        },
+                    },
+                },
+            }
+        ]
+    }
+
     var workspace = Blockly.inject('blocklyDiv',
         {
```
Comment (BeksOmega) [inline examples/blockly-umd/index.js]: If you could also fix the indentation for the rest of this file that would be great! (There might be a way to automatically convert it depending on the IDE you're using)

**Review by BeksOmega — CHANGES_REQUESTED** [review]
LGTM! Thank you for your work on this :D Just a few nit-picky whitespace changes hehe. (Sorry the linter in samples doesn't always catch everything!)

File: examples/blockly-umd/index.js (hunk 2 above)
Comment (richardla4) [reply examples/blockly-umd/index.js]: Ok, I'll make those changes!

**Review by BeksOmega — APPROVED** [review]
Sweet! I'm just going to let the CI (continuous integration - meaning automated testing) run and then get this merged!

**Pull request merged**